--- mkspecs/macx-g++/qmake.conf	2007-02-02 15:01:44.000000000 +0100
+++ modif/qmake.conf	2010-11-02 12:41:26.000000000 +0100
@@ -18,7 +18,7 @@
 QMAKE_YACC_HEADER       = $base.tab.h
 QMAKE_YACC_SOURCE       = $base.tab.c
 QMAKE_RESOURCE		= /Developer/Tools/Rez
-QMAKE_CFLAGS		= -pipe
+QMAKE_CFLAGS		= -isysroot /Developer/SDKs/MacOSX10.6.sdk -pipe -m32 -arch i386
 QMAKE_CFLAGS_DEPS	= -M
 QMAKE_CFLAGS_WARN_ON	= -Wall -W
 QMAKE_CFLAGS_WARN_OFF	= -w
@@ -46,7 +46,7 @@
 
 QMAKE_LINK		= $$QMAKE_CXX
 QMAKE_LINK_SHLIB	= $$QMAKE_CXX
-QMAKE_LFLAGS		= -headerpad_max_install_names
+QMAKE_LFLAGS		= -arch i386 -headerpad_max_install_names
 #QMAKE_LFLAGS_PREBIND    = -seg1addr 0x90000000
 QMAKE_LFLAGS_RELEASE	=
 QMAKE_LFLAGS_DEBUG	=
